docs/language/README.md

```diff
@@ -90,11 +90,12 @@ as in the shorter form
 ```
 count() by color
 ```
-Similarly, the canonical form of a search expression is a `filter` operator
-(and the "AND" operator is explicit in canonical form),
-so the example from above would be written canonically as
+Similarly, the canonical form of a search expression includes a `filter`
+operator and the searched word enclosed in a `match()` function (also, the "AND"
+operator is explicit in canonical form). Therefore the example from above would
+be written canonically as
 ```
-filter widget and price > 1000
+filter match("widget") and price > 1000
```

Comment:
`zc -C` shows the `match()`, so I've opted to include it here and on the other relevant places on the page. I'm not sure if its exclusion was intentional in attempt to keep things looking simpler. However, canonical form is mentioned enough times on this page that I felt like we might as well go all the way with it.